oaisys.gemspec

```diff
@@ -26,6 +26,8 @@ Gem::Specification.new do |spec|
   spec.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
   spec.add_dependency "rails", "~> 5.2.3"
+  spec.add_dependency 'builder', '~> 3.0'
 
   spec.add_development_dependency "pg"
+  spec.add_development_dependency "byebug"
```

Comment:
This is just me being fussy, but how strongly do you feel about byebug over pry? I tend to prefer pry, personally.

Reply:
I've been using byebug but willing to try new things, especially if you think pry is superior. Why do you prefer it?

Reply:
byebug is more of a debugger, and pry is more of a full-on on-demand REPL. pry is less useful for stepping through code (which I don't tend to do that often) and more useful for doing a lot of exploratory programming at a given breakpoint and then continuing with execution.

I guess give it a try and see how you feel about it. Worst case if you want more of the debugging features, `pry-byebug` will add them to pry